core/data/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/data/sessions/FakeSessionsApiClient.kt

```diff
@@ -73,42 +73,44 @@ fun SessionsAllResponse.Companion.fake(): SessionsAllResponse {
 
     for (day in 0 until 3) {
         val dayOffset = day * 24 * 60 * 60
-        for (index in 0 until 4) {
-            val start = Instant.fromEpochSeconds(
-                LocalDateTime.parse("2023-09-14T10:10:00")
-                    .toInstant(TimeZone.of("UTC+9")).epochSeconds + index * 25 * 60 + dayOffset,
-            ).toLocalDateTime(TimeZone.of("UTC+9"))
-            val end = Instant.fromEpochSeconds(
-                LocalDateTime.parse("2023-09-14T10:50:00")
-                    .toInstant(TimeZone.of("UTC+9")).epochSeconds + index * 25 * 60 + dayOffset,
-            ).toLocalDateTime(TimeZone.of("UTC+9"))
+        for (room in 0 until 5) {
```

Comment:
Sorry but we might have to add rooms here 👀 

```
    val rooms = listOf(
        RoomResponse(name = LocaledResponse(ja = "Room1", en = "Room2"), id = 1, sort = 1),
    )
```

Reply:
Thank you!
I confirmed that unit tests pass in my local environment.